graylog2-server/src/main/java/org/graylog2/rest/resources/system/contentpacks/ContentPackResource.java

```diff
@@ -242,6 +246,12 @@ public void deleteContentPack(
             @ApiParam(name = "revision", value = "Content Pack revision", required = true)
             @PathParam("revision") final int revision) {
         checkPermission(RestPermissions.CONTENT_PACK_DELETE, contentPackId.toString());
+
+        if (contentPackInstallationPersistenceService.findByContentPackIdAndRevision(contentPackId, revision).size() > 0) {
```

Comment:
Small suggestion, as above:

```suggestion
        if (!contentPackInstallationPersistenceService.findByContentPackIdAndRevision(contentPackId, revision).isEmpty()) {
```